tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -840,42 +859,26 @@ def forward(
                     f"  M={m}, N={n}, K={k}\n"
                     f"  Allowed backends: {self.allowed_backends}")
 
-        if tactic == "cuda_core":
+        backend, sub_tactic = tactic
+        if backend == "cuda_core":
             # Unswizzle the activation scale factors
             # act_sf is swizzled, need to reverse it for cuda_core_nvfp4_gemm
             m = act_fp4.shape[0]
             act_sf_unswizzled = torch.ops.trtllm.block_scale_interleave_reverse(
```

Ruff (0.14.10)

Local variable `act_sf_unswizzled` is assigned to but never used

Remove assignment to unused variable `act_sf_unswizzled`

(F841)